config/config.go

```diff
@@ -170,23 +170,23 @@ func (b BaseConfig) ChainID() string {
 }
 
 // GenesisFile returns the full path to the genesis.json file
-func (b BaseConfig) GenesisFile() string {
-	return rootify(b.Genesis, b.RootDir)
+func (c BaseConfig) GenesisFile() string {
+	return rootify(c.Genesis, c.RootDir)
 }
 
 // PrivValidatorFile returns the full path to the priv_validator.json file
-func (b BaseConfig) PrivValidatorFile() string {
-	return rootify(b.PrivValidator, b.RootDir)
+func (c BaseConfig) PrivValidatorFile() string {
+	return rootify(c.PrivValidator, c.RootDir)
 }
 
 // NodeKeyFile returns the full path to the node_key.json file
-func (b BaseConfig) NodeKeyFile() string {
-	return rootify(b.NodeKey, b.RootDir)
+func (c BaseConfig) NodeKeyFile() string {
+	return rootify(c.NodeKey, c.RootDir)
 }
 
 // DBDir returns the full path to the database directory
-func (b BaseConfig) DBDir() string {
-	return rootify(b.DBPath, b.RootDir)
+func (c BaseConfig) DBDir() string {
```

Comment:
why `cfg` below, but here you are using `c`? should not we be consistent? i.e. `cfg` everywhere

Reply:
`BaseConfig` type was using `b` and `c` as receiver names and `ConsensusConfig` type was using `c` and `cfg`. I tried to minimize changes by settling on `c` for `BaseConfig` and on `cfg` for `ConsensusConfig`. I will change to `cfg` for both types.